.github/workflows/pkg_msi.yaml

```diff
@@ -158,18 +170,80 @@ jobs:
 
       - name: Cleanup dist before upload
         run: |
-          Remove-Item -Path .\dist\cnquery.exe -Force
-          Remove-Item -Path .\dist\cnspec.exe -Force
+          Remove-Item -Path .\dist\${{ matrix.arch }}\cnquery.exe -Force
+          Remove-Item -Path .\dist\${{ matrix.arch }}\cnspec.exe -Force
 
       - name: Upload Distribution
         uses: actions/upload-artifact@v4
         with:
-          name: msi
-          path: dist/
+          name: msi-${{ matrix.arch }}
+          path: dist/${{ matrix.arch }}
+
+  test-msi-install:
+    name: Test Signed Package
+    strategy:
+      matrix:
+        arch:
+          - amd64
+#          - arm64 # we currently don't have windows arm64 runners to test arm against
+    needs:
+      - setup
+      - msi-build
+    runs-on: windows-latest
+    steps:
+      - name: Download MSI Package
+        uses: actions/download-artifact@v4
+        with:
+          name: msi-${{ matrix.arch }}
+          path: dist
+      - name: Verify digital signature is valid
+        run: |
+          cd dist
+          Get-AuthenticodeSignature -FilePath .\\mondoo_${{ matrix.arch }}.msi
+      - name: Install artifact
+        run: |
+          cd dist
+          msiexec /qn /i mondoo_${{ matrix.arch }}.msi
+      - name: Verify the correct cnquery version is installed
+        run: |
+          $version=& 'C:\Program Files\Mondoo\cnquery.exe' version
+          $match=$version -like "*${{ needs.setup.outputs.version }}*"
+          if (-not $match) {
+            exit 1
+          }
+      - name: Run a basic cnquery sanity check
+        run: |
+          & 'C:\Program Files\Mondoo\cnquery.exe' run -c "os.base.packages.where(name == 'Mondoo') { name }"
+      - name: Verify the correct cnspec version is installed
+        run: |
+          $version=& 'C:\Program Files\Mondoo\cnspec.exe' version
+          $match=$version -like "*${{ needs.setup.outputs.version }}*"
+          if (-not $match) {
+            exit 1
+          }
+      - name: Run a basic cnquery sanity check
```

Comment:
Do you mean to test cnspec here?

Reply:
yes, fixed in https://github.com/mondoohq/installer/pull/491